ui/app/peers/[peerName]/datatables.tsx

```diff
@@ -0,0 +1,112 @@
+import { CopyButton } from '@/components/CopyButton';
+import { SlotInfo, StatInfo } from '@/grpc_generated/route';
+import { Label } from '@/lib/Label';
+import { Table, TableCell, TableRow } from '@/lib/Table';
+import Link from 'next/link';
+
+export const SlotTable = ({ data }: { data: SlotInfo[] }) => {
+  const getFlowName = (slotName: string) => {
+    if (slotName.startsWith('peerflow_slot_')) {
+      return slotName.slice(14);
+    }
+    return '';
+  };
+  return (
+    <div style={{ height: '30%', marginTop: '2rem' }}>
+      <div style={{ fontSize: 17, marginBottom: '1rem' }}>
+        Replication Slot Information
+      </div>
+      <div style={{ maxHeight: '100%', overflow: 'scroll' }}>
+        <Table
+          header={
+            <TableRow>
+              <TableCell as='th'>Slot Name</TableCell>
+              <TableCell as='th'>Redo LSN</TableCell>
+              <TableCell as='th'>Restart LSN</TableCell>
+              <TableCell as='th'>Lag (In MB)</TableCell>
+            </TableRow>
+          }
+        >
+          {data.map(({ slotName, redoLSN, restartLSN, lagInMb }) => {
+            const flowName = getFlowName(slotName);
+            return (
+              <TableRow key={slotName}>
+                <TableCell>
+                  {flowName.length >= 1 ? (
```

Comment:
refactor to its own component function `const SlotMirrorInfo = .. { slotName, flowName?}` or something along those lines.